crates/relayer-components/src/relay/impls/connection/mod.rs

```diff
@@ -0,0 +1,6 @@
+pub mod bootstrap;
```

Comment:
```suggestion
//! This module contains the traits and types necessary in order to perform the
//! [IBC connection handshake protocol](https://github.com/cosmos/ibc/tree/main/spec/core/ics-003-connection-semantics) 
//! between two IBC-enabled blockchains. The protocol establishes a connection 
//! between the two participating blockchains. This connection between the two 
//! chains is necessary in order for channels to be established between them.
//!
//! Each of the different `open_*` modules, except for `open_handshake`, defines 
//! a component responsible for a single step in the handshake protocol. The 
//! `open_handshake` module wires together the steps that actually constitute
//! the handshake protocol proper. The `open_init` step is not strictly part
//! of the handshake; it merely signals the start of the handshake process. 
//! The `bootstrap` module is the highest level module that wires together the
//! initialization and handshake steps.

pub mod bootstrap;
```

Reply:
Technically, the bootstrap function is only used when it is initiated by the relayer user or inside tests. If the connection is initiated by external users, then only the `open_handshake` component is used.